x-pack/platform/test/api_integration/apis/ml/model_management/nodes_overview.ts

```diff
@@ -75,7 +75,7 @@ export default ({ getService }: FtrProviderContext) => {
       expect(body.nodes[0].roles).to.contain('ml');
       expect(body.nodes[0].memory_overview.anomaly_detection.total).to.be.greaterThan(10000000);
       expect(body.nodes[0].memory_overview.dfa_training.total).to.eql(0);
-      expect(body.nodes[0].memory_overview.trained_models.total).to.eql(0);
+      expect(body.nodes[0].memory_overview.trained_models.total).to.be.greaterThan(10000000);
```

Comment:
The product docs are installed by default in security, so the elser model is installed